### PR TITLE
chore: update the readme path in the docker ci

### PR DESCRIPTION
**Pull request by ayushnau**
Changes Added.
1. updated the readme.md path in the docker release ci.
2. Added the git checkout action in the docker ci.

Related issue(s)

Related to: https://github.com/asyncapi/generator/issues/1044

### CODE REVIEW

File: .github/workflows/release-docker.yml [hunk 1]
```diff
@@ -10,6 +10,9 @@ jobs:
     name: Docker build and push
     runs-on: ubuntu-latest
     steps:
+      - name: Check out the repository
```
Comment (derberg) [inline .github/workflows/release-docker.yml:13]: make sure it runs only before the step related to readme and use the latest checkout action

Comment (ayushnau) [reply .github/workflows/release-docker.yml:13]: done please check.

Comment (derberg) [reply .github/workflows/release-docker.yml:13]: now you do checkout twice, and still with old version of the action

**Review by derberg — APPROVED** [review]

**Pull request merged**